kafka-connect-aws-s3/src/it/scala/io/lenses/streamreactor/connect/aws/s3/sink/S3SinkTaskJsonEnvelopeTest.scala

```diff
@@ -173,7 +173,7 @@ class S3SinkTaskJsonEnvelopeTest
     val props = DefaultProps
       .combine(
         Map(
-          "connect.s3.kcql" -> s"insert into $BucketName:$PrefixName select * from $TopicName STOREAS `JSON`  WITH_FLUSH_COUNT = 3 PROPERTIES('store.envelope'=true)",
+          "connect.s3.kcql" -> s"insert into $BucketName:$PrefixName select * from $TopicName STOREAS `JSON`  WITH_FLUSH_COUNT = 3 PROPERTIES('store.envelope'=true,'padding.fields.enable'='partition,offset')",
```

Comment:
it might be clear and less error-prone to have

padding.partition=true
padding.offset=true?

Reply:
If we're going to do this, why not use it to set the padding length instead?  It seems like we should be able to have different fields with different padding lengths.

```
padding.partition=4
padding.offset=12
```

Reply:
Indeed. Nice suggestion

Reply:
Thanks - I've implemented these as 

`padding.length.partition` and `padding.length.offset` (and implemented in a future-proof way that will not rule out allowing padding on custom fields)